src/components/Image/Image.visual.tsx

```diff
@@ -131,6 +132,45 @@ visualize('Image', () => {
           />
         ));
       });
+
+      story('with wiring', () => {
```

Comment:
I really liked this approach of testing the wiring, maybe we would like to make it a convention?

Reply:
Yes, I also think that we should add this kind of tests for all the components :)